fix: Add example Injection Protection plugin log (#8286)
Kong/docs.konghq.com: app/_hub/kong-inc/injection-protection/overview/_index.md

    * [UDP Log](/hub/kong-inc/udp-log/)
* [{{site.konnect_short_name}} Advanced Analytics](/konnect/analytics/) <!--though i'm not sure if they will do anything with them yet--> 

> The following is an example of the information that is logged by the Injection Protection plugin:

Reviewer: ```suggestion
Here's a sample log entry created by the Injection Protection plugin. 
```
Slight rephrase to make it sound less formal + shorten the sentence.

```
threat detected: 'sql', action taken: log_only, found in path_and_query, query param value: foo: insert into test
```

> 

Reviewer: ```suggestion
Where:
* The plugin detected a SQL injection threat
* It created a log entry but took no other actions
* The threat was found in the request path or query
* The inserted snippet is `foo: insert into test`
```
I'd add something like this to give the log message a tiny bit of context. Not sure if I got it all correct, so please double-check.
